### PR TITLE
S02 us04form recipe app step

### PR DESCRIPTION
**Pull request by plbonnier**

### CODE REVIEW

**Review by nickyrasa — APPROVED** [review]

**Review by Thomas-Lunardo — APPROVED** [review]

File: assets/app.js [hunk 1]
```diff
@@ -10,3 +10,5 @@ import './styles/app.scss';
 
 // start the Stimulus application
 import './bootstrap';
+
+console.log('toto vas-y!!!')
```
Comment (StoicFrenchDev) [inline assets/app.js:14]: Is this necessary?

File: src/Form/RecipeType.php [hunk 2]
```diff
@@ -4,31 +4,53 @@
...
+                'label' => 'Temps de préparation : '
+            ])
+            ->add('personNumber', NumberType::class, [
+                'label' => 'Pour combien de personne ? '
+            ])
+            ->add('steps', CollectionType::class, [
+                'entry_type' => StepType::class,
+                'entry_options' => ['label' => false],
+                'allow_add' => true,
+                // 'allow_delete' => true,
```
Comment (StoicFrenchDev) [inline src/Form/RecipeType.php:50]: Dead code

File: templates/home/index.html.twig [hunk 3]
```diff
@@ -10,7 +10,8 @@
       <div class="carousel-inner">
         {% for key, recipe in showRecipes %}
           <div class="carousel-item {% if key == 0 %} active {% endif %}">
-           <a href="{{ path('app_recipe_show', { id:recipe.id }) }}"> <img src="{{ asset('build/images/' ~ recipe.picture) }}" class="pic-recipe object-fit-cover" alt="{{ recipe.nameRecipe }}" /></a>
+          <a href="{{ path('app_recipe_show', { id:recipe.id }) }}"> <img src="{{ asset('build/images/' ~ recipe.picture) }}" class="pic-recipe object-fit-cover" alt="{{ recipe.nameRecipe }}" /></a>
+          {# {{ asset('/uploads/images/pictures/' ~ recipe.picture) }} quand on enregistre par le formulaire Ben ne le dégage pas #}
```
Comment (StoicFrenchDev) [inline templates/home/index.html.twig:14]: Hmmmmmmmmmmmmmmmmmmmmm

File: templates/recipe/new.html.twig [hunk 4]
```diff
@@ -1,15 +1,22 @@
...
+
+		<h1 class="mt-2 mb-2 text-center">Ajouter une recette</h1>
+					
+				<div class="container-add-form">
+		<img src="/build/images/nomnomlogo.png" alt="logo de nomnomnom" class="logo-add">
+		</div>
+				<div class="paragraph mb-3">
+			<div>{{ include('recipe/_form.html.twig') }}<div>
+
+		{# <a class="mb-4" href="{{ path('app_recipe_index') }}">Retour à la liste des recettes</a> #}
```
Comment (StoicFrenchDev) [inline templates/recipe/new.html.twig:17]: Dead code

File: src/Form/RecipeType.php (hunk 2 above)
Comment (plbonnier) [reply src/Form/RecipeType.php]: delete done

File: templates/recipe/new.html.twig (hunk 4 above)
Comment (plbonnier) [reply templates/recipe/new.html.twig:17]: change done

File: templates/home/index.html.twig (hunk 3 above)
Comment (plbonnier) [reply templates/home/index.html.twig]: delete done

File: templates/recipe/new.html.twig [hunk 5]
```diff
@@ -1,15 +1,20 @@
...
-{% block title %}New Recipe
+{% block title %}Ajouter une recette
 {% endblock %}
 
 {% block body %}
-	<div class="container mb-5">
-		<h1 class="mt-5 mb-5 text-center">Create new Recipe</h1>
-		<div class="form-control text-center fs-3 mb-3 p-5">
-			{{ include('recipe/_form.html.twig') }}
+	<div class="container-fluid w-100 mt-4 px-4 paragraph">
```
Comment (StoicFrenchDev) [inline templates/recipe/new.html.twig:7]: unclosed div tag

File: templates/recipe/new.html.twig [hunk 6]
```diff
@@ -1,15 +1,20 @@
...
+
+		<h1 class="mt-2 mb-2 text-center">Ajouter une recette</h1>
+					
+				<div class="container-add-form">
+		<img src="/build/images/nomnomlogo.png" alt="logo de nomnomnom" class="logo-add">
+		</div>
+				<div class="paragraph mb-3">
+			<div>{{ include('recipe/_form.html.twig') }}<div>
```
Comment (StoicFrenchDev) [inline templates/recipe/new.html.twig:15]: is this supposed to be a closing tag?

**Review by StoicFrenchDev — CHANGES_REQUESTED** [review]

**Review by nickyrasa — APPROVED** [review]

**Review by Aude711 — APPROVED** [review]

**Review by Thomas-Lunardo — APPROVED** [review]

**Review by AminaChakirDev — APPROVED** [review]

**Review by StoicFrenchDev — APPROVED** [review]

**Pull request merged**